javav2/example_code/ses/src/main/java/com/example/sesv2/SendMessageAttachment.java

```diff
@@ -0,0 +1,166 @@
+// snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
+// snippet-sourcedescription:[SendMessageAttachment.java demonstrates how to send an email message that contains an attachment by using the SesV2Client.]
+// snippet-keyword:[AWS SDK for Java v2]
+// snippet-keyword:[Amazon Simple Email Service]
+// snippet-keyword:[Code Sample]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[05/19/2022]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+
+package com.example.sesv2;
+
+// snippet-start:[ses.java2.sendmessage.request.sesv2.import]
+import software.amazon.awssdk.auth.credentials.ProfileCredentialsProvider;
+import software.amazon.awssdk.core.SdkBytes;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.sesv2.model.RawMessage;
+import software.amazon.awssdk.services.sesv2.SesV2Client;
+import software.amazon.awssdk.services.sesv2.model.EmailContent;
+import software.amazon.awssdk.services.sesv2.model.SendEmailRequest;
+import software.amazon.awssdk.services.sesv2.model.SesV2Exception;
+import javax.activation.DataHandler;
+import javax.activation.DataSource;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+import javax.mail.util.ByteArrayDataSource;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.file.Files;
+import java.util.Properties;
+// snippet-end:[ses.java2.sendmessage.request.sesv2.import]
+
+public class SendMessageAttachment {
+
+        public static void main(String[] args) throws MessagingException, IOException {
+
+                final String usage = "\n" +
+                        "Usage:\n" +
+                        "    <sender> <recipient> <subject> \n\n" +
+                        "Where:\n" +
+                        "    sender - An email address that represents the sender. \n"+
+                        "    recipient -  An email address that represents the recipient. \n"+
+                        "    subject - The  subject line. \n" ;

```

Comment:
Remove extra space between "The" and "subject".

Reply:
fixed